lib/ansible/modules/windows/win_find.ps1

```diff
@@ -303,7 +307,10 @@ Function Get-FilesInFolder($path) {
     try {
         $dir_files = $dir.EnumerateFileSystemInfos("*", [System.IO.SearchOption]::TopDirectoryOnly)
     } catch [System.IO.DirectoryNotFoundException] { # Broken ReparsePoint/Symlink, cannot enumerate
-    } catch [System.UnauthorizedAccessException] {}  # No ListDirectory permissions, Get-ChildItem ignored this
+    	Add-Warning -obj $result -message "symlink enumeration error: $($_.Exception.Message)"
+    } catch [System.UnauthorizedAccessException] {  # No ListDirectory permissions, Get-ChildItem ignored this
+        Add-Warning -obj $result -message "symlink enumeration error: $($_.Exception.Message)"
```

Comment:
I would leave this as is for now. We need to look more into win_find and how it enumerates but prefer to do that sometime in the future without adding more warnings for now.